src/item.cpp

```diff
@@ -2336,13 +2345,15 @@ int item::get_warmth() const
     int result = static_cast<int>( t->warmth );
 
     if (item::item_tags.count("furred") > 0){
-        warmed = 2 * volume() * (float(get_coverage()) / 100);
-        return result + warmed;
-    } else {
-        return result;
+        fur_lined = 2 * volume() * (float(get_coverage()) / 100);
+    }
+    if (item::item_tags.count("wooled") > 0){
+        wool_lined = 1.5 * volume() * (float(get_coverage()) / 100);
```

Comment:
Ahem.  Volume doesn't make much sense in this context.  No need to keep perpetuating the last time I communicated poorly.  :-(